eth/transactionManager_test.go

```diff
@@ -209,6 +209,7 @@ func TestTransactionManager_Replace(t *testing.T) {
 	)
 
 	// Error signing replacement tx
+	gpm.maxGasPrice = nil
```

Comment:
This line doesn't actual change the behavior of this test because the maxGasPrice is set again on L219 before the `replace()` call. So, if you revert the update in `transactionManager.go`, all tests would still pass.

I also recommend creating a separate test case specifically for this nil pointer error as opposed to modifying the state that an existing test case depends on to make it easier to identify the state that is relevant for the test case. For example:

```diff
diff --git a/eth/transactionManager_test.go b/eth/transactionManager_test.go
index aee3f34c..37c20d6c 100644
--- a/eth/transactionManager_test.go
+++ b/eth/transactionManager_test.go
@@ -209,7 +209,6 @@ func TestTransactionManager_Replace(t *testing.T) {
        )

        // Error signing replacement tx
-       gpm.maxGasPrice = nil
        expErr = errors.New("SignTx error")
        sig := &stubTransactionSigner{
                err: nil,
@@ -220,6 +219,13 @@ func TestTransactionManager_Replace(t *testing.T) {
        tx, err = tm.replace(stubTx)
        assert.Nil(tx)
        assert.EqualError(err, expErr.Error())
+
+       // Test when max gas price is nil - should still return signing replacement tx error
+       gpm.maxGasPrice = nil
+       tx, err = tm.replace(stubTx)
+       assert.Nil(tx)
+       assert.EqualError(err, expErr.Error())
+
        sig.err = nil

        // Error sending replacement tx
```

Without the update in `transactionManager.go` this new test case should cause tests to fail due to a nil pointer error and with the update all tests should pass.